.github/workflows/ci.yaml

```diff
@@ -0,0 +1,172 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  TESTS: "FunctionTests SDKTests RoomTests PublishDataTests PublishBufferCapturerTests VideoViewTests TrackTests"
```

Comment:
is this really necessary? seems like an easy trap that we'll add new tests but forget to add them here and thus CI won't actually run them

Reply:
This part was copied from the existing test workflow.

Reply:
Yes Ben is right, but some tests don't run on CI/Simulator needs device etc. We can organize this in another PR.